app/controllers/comments_controller.rb

```diff
@@ -12,6 +12,13 @@ def create
     end
   end
 
+  def destroy
```

Comment:
- [ ] In order use  **CanCanCan** for Comment authorization in `Comment Controller`, add this line `load_and_authorize_resource` on the top of your controller 👇🏻 

   ```ruby
     class CommentsController < ApplicationController
       load_and_authorize_resource
   
       ...
     end
   ```